t/t.go

```diff
@@ -1006,9 +1011,6 @@ func run() error {
 		if testSuiteContains("load") || testSuiteContains("all") {
 			downloadDataFiles()
```

Comment:
will we have the same problem if this function fails too?

Reply:
Resolved: Applied the same fix for the load test.